df-logger/lang/es.json

```diff
@@ -43,7 +70,7 @@
 			"21": "Se acerca el Challenger - ¡{{username}} ha aparecido!",
 			"22": "¿Es un pájaro? ¿Es un avión? Nah, olvídalo, solo es {{username}}.",
 			"23": "¡Es {{username}}! ¡Alabado sea el Sol!\n\\\\[T]/",
-			"24": "Nunca perderé la confianza en {{username}}. Nunca decepcionaré a {{username}}.",
+			"24": "{{username}} es un honrado traficante\nTomando copas me lo encuentro to' los días\nMe cuenta historias de sus años en la cárcel\nA veces había suerte y si tenía pasta salía",
```

Comment:
I think this is a personal message. Supposed to be a line from the song "Never Gonna Give You Up".